code/datums/uplink/announcements.dm

```diff
@@ -31,7 +31,7 @@
 /datum/uplink_item/abstract/announcements/fake_crew_arrival
 	name = "Crew Arrival Announcement/Records"
 	desc = "Creates a fake crew arrival announcement as well as fake crew records, using your current appearance (including held items!) and worn id card. Trigger with care!"
-	item_cost = 8
+	item_cost = 3
```

Comment:
The current price of that is fine, as you shouldn't be able to disguise your entire merc team as crew and still have enough TCs to purchase potent weapons from the uplink

Reply:
yee